superset/db_engine_specs/elasticsearch.py

```diff
@@ -123,7 +123,7 @@ def convert_dttm(
         sqla_type = cls.get_sqla_column_type(target_type)
 
         if isinstance(sqla_type, types.DateTime):
-            return f"""'{dttm.isoformat(timespec="seconds")}'"""
+            return f"""'{dttm.isoformat(sep=" ", timespec="seconds")}'"""
```

Comment:
@hhhonzik would you mind adding some unit tests for this? You can find examples of these with the other DB engine specs.

Reply:
@john-bodley Sorry i ignored them in the first place. Turns out there is a test for this and i didn't update it.